dietpi/dietpi-software

```diff
@@ -11775,21 +11769,7 @@ _EOF_
 					then
 						if [[ ! -f '/mnt/dietpi_userdata/radarr/config.xml' ]]
 						then
```

Comment:
Actually even this initial existence check could be added to the function as well, so:
1. Check once if file and in case content is present, before starting the service. If so, `return 0` directly.
2. Start service, wait for file and in case content.
3. Final check if file and content is present.

It's three times the relatively long check block in the function, but overall code is reduced most, it is always both checked, file and content. Currently, if the file exist prior to install, but e.g. it's an old version or the service was started but terminated quickly, the content could be missing, leading to the following configuration steps failing.